### PR TITLE
[EDA-599] Display notification messages

### PR DESCRIPTION
**Pull request by nicolasnunez-eb**
When a functionality is executed there must be a success message or an error message

### CODE REVIEW

File: supplier_app/__init__.py [hunk 1]
```diff
@@ -4,6 +4,14 @@
 
 DBTuple = namedtuple('DBTuple', 'value verbose_name')
 
+EMAIL_SUCCESS_MESSAGE = "Your invitation was sent successfully"
+EMAIL_ERROR_MESSAGE = "There was an error sending your email"
+
+TAXPAYER_CREATION_SUCCESS_MESSAGE = "Your taxpayer was created successfully"
+TAXPAYER_CREATION_ERROR_MESSAGE = "There was an error on creating your taxpayer please try again"
+TAXPAYER_FORM_INVALID_MESSAGE = "Your form is invalid"
+COMPANY_ERROR_MESSAGE = "You don't have a company yet please contact with us 'mail@mail.com'"
```
Comment (leandro-eb) [inline supplier_app/__init__.py]: Are we aware of `i18n` for this?

File: supplier_app/tests/__init__.py [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
 )
 from supplier_app.bank_info import BANK_INFO
 
+BUSINESS_EXAMPLE_NAME_1 = 'Pyme 1'
+BUSINESS_EXAMPLE_NAME_2 = 'Pyme 2'
 
 STATUS_PENDING = "PENDING"
 STATUS_ACTIVE = "ACTIVE"
 STATUS_CHANGE_REQUIRED = "CHANGE REQUIRED"
 STATUS_DENIED = "DENIED"
```
Comment (leandro-eb) [inline supplier_app/tests/__init__.py]: This could be moved to a constant file

File: supplier_app/tests/test_forms.py [hunk 3]
```diff
@@ -22,7 +23,7 @@
 
 from supplier_app.tests import (
     taxpayer_creation_POST_factory,
-    get_bank_info_example
+    get_bank_info_example,
 )
```
Comment (leandro-eb) [inline supplier_app/tests/test_forms.py]: We should consider using isort for all dependencies

File: supplier_app/tests/test_view.py [hunk 4]
```diff
@@ -151,13 +157,16 @@ def test_get_success_url_should_redirect_to_home(self):
             )
 
     def _get_taxpayer_creation_request(self):
-        request = self.factory.get(
+        request = self.factory.post(
```
Comment (leandro-eb) [inline supplier_app/tests/test_view.py]: should we consider changing the name of the function?

File: supplier_app/views.py [hunk 5]
```diff
@@ -266,21 +285,26 @@ def approve_taxpayer(self, taxpayer_id, request=None):
...
-    send_email_notification(subject, body, email)
-    return redirect('company-list')
+def company_invite(request, **kwargs):
+    try:
+        email = [request.POST['email']]
+        company_id = request.POST['company_id']
+        company = Company.objects.get(pk=company_id)
+        companyuniquetoken = CompanyUniqueToken(company=company)
+        companyuniquetoken.assing_company_token
+        companyuniquetoken.save()
```
Comment (leandro-eb) [inline supplier_app/views.py]: please, use `_` on this variable

File: supplier_app/__init__.py (hunk 1 above)
Comment (nicolasnunez-eb) [reply supplier_app/__init__.py]: For now they are only available in english, I think when we have to add another language we have to add new special messages for that language.
Do you think I have to make a function(where it receive a language and then he return me the message for that language) now or I have to consider for the future?

File: supplier_app/tests/test_forms.py (hunk 3 above)
Comment (nicolasnunez-eb) [reply supplier_app/tests/test_forms.py]: Some people of the team has already implemented, I will try to implement it

File: supplier_app/views.py (hunk 5 above)
Comment (nicolasnunez-eb) [reply supplier_app/views.py]: Why we have to use _ ?

File: supplier_app/tests/test_forms.py (hunk 3 above)
Comment (nicolasnunez-eb) [reply supplier_app/tests/test_forms.py]: resolved

File: supplier_app/tests/test_view.py [hunk 6]
```diff
@@ -1080,3 +1093,79 @@ def test_change_taxpayer_status_to_denied_sends_email_notification(self):
...
+        response = self._make_email_invitation_post()
+        self.assertContains(response, EMAIL_SUCCESS_MESSAGE)
+
+    @patch(
+        'supplier_app.views.send_email_notification',
+        side_effect=Exception
+    )
+    @patch(
+        'supplier_app.models.CompanyUniqueToken._token_generator',
+        return_value='f360da6197be4436a4b686460289085c14a859d634a9daca2d7d137b178b193e'
```
[the remark this answers is not in the document]
Comment (nicolasnunez-eb) [reply supplier_app/tests/test_view.py]: resolved

File: supplier_app/tests/test_view.py (hunk 4 above)
Comment (nicolasnunez-eb) [reply supplier_app/tests/test_view.py]: I change it but I don't know what is wrong with the name

File: supplier_app/views.py (hunk 5 above)
Comment (nicolasnunez-eb) [reply supplier_app/views.py]: resolved

**Pull request merged**